tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -258,6 +258,11 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 					return false
 				}
 
+				if utils.PipelineRunFailed(releasePr) || utils.PipelineRunFailed(additionalReleasePr) {
+					GinkgoWriter.Printf("\n One of two PipelineRuns failed: %s   %s", releasePr.Name, additionalReleasePr.Name)
+					return false
```

Comment:
the condition is correct, but returning `false` doesn't fail the test immediately, which is what we want in this case. gomega assertion is needed here. see how it's done in other tests, e.g. [mvp-demo](https://github.com/redhat-appstudio/e2e-tests/blob/1348d11629a5dcede1d0681fe43ffaad25ddfdc1/tests/mvp-demo/mvp-demo.go#L514)